docs/finality-provider-operation.md

```diff
@@ -731,7 +731,136 @@ The response includes:
+eligible to receive rewards.
+
+```shell
+fpd withdraw-reward <type> --from <registered-bbn-address>
+--keyring-backend test --home <home-dir> --fees <fees>
+```
+
+Parameters:
+
+* `<type>`: The type of reward to withdraw (one of `finality_provider`,
+  `btc_delegation`)
+* `--from`: The finality provider's registered Babylon address.
+* `<registered-bbn-address>`: Address rewards are withdrawn to.
+* `--keyring-backend`: The keyring backend to use.
+* `--home`: The home directory for the finality provider.
+* `--fees`: The fees to pay for the transaction, should be over `400ubbn`.
+  These fees are paid from the account specified in `--from`.
+
+Again, this command should ask to
+`confirm transaction before signing and broadcasting [y/N]:` and output the
+transaction hash.
+
+This will withdraw **ALL** accumulated rewards to the address you set in the
+`set-withdraw-addr` command if you set one. If no withdrawal address was set,
+the rewards will be withdrawn to your finality provider address.
+
+#### 6.2.3 Set Withdraw Address
+
+To set the withdraw address to the beneficiary key, use the following command:
+
+```shell
+fpd set-withdraw-addr <beneficiary-address> --from <registered-bbn-address>
+--keyring-backend test --home <home-dir> --fees <fees>
+```
+
+Parameters:
+
+* `<beneficiary-address>`: Corresponds to the beneficiary key and is where
+  withdraw rewards are sent to.
+* `<registered-bbn-address>`: Corresponds to the key used in registration and is where
+  withdraw rewards are sent to by default if no other address is set via `set-withdraw-addr`
+* `--from`: The finality provider's registered Babylon address.
+* `--keyring-backend`: The keyring backend to use.
+* `--home`: The home directory for the finality provider.
+* `--fees`: The fees to pay for the transaction, should be over `400ubbn`.
+  These fees are paid from the account specified in `--from`.
+
+This command should ask to
+`confirm transaction before signing and broadcasting [y/N]:` and output the
+transaction hash.
+
+### 6.2.4 Refunding finality provider
+
+To support the gas costs associated with committing randomness, which are not
+refunded by the protocol, we recommend setting up a refunding flow.
+This involves periodically transferring rewards from a beneficiary key to the
+operational key used by the finality provider.
+
+#### How to set up the refunding process
+
+1. Ensure you have two keys:
+   * Beneficiary Key: Receives staking rewards.
+   * Operational Key: Used by the finality provider daemon to submit transactions.
+
+  If this step hasnt been completed, follow the steps in
+  [5.2 Add key for the Babylon account](#52-add-key-for-the-babylon-account) and
+  create 2 additional keys.
+
+2. **Configure Withdrawals**:
+  Ensure the withdraw address is set to the beneficiary key using the
+  `set-withdraw-addr` command.
+
+  This sets the withdraw address to the beneficiary key to enable
+  withdrawing rewards periodically.
```

Comment:
Don't think this sentence is necessary. This basically repeats the last sentence and `periodically` part should be done by a cron job